### ISSUE TITLE
Minor manuscript suggestions 

### ISSUE THREAD

**Issue by chrisleaman** [npucino/sandpyper#13]
Hi @npucino, please see below for my comments on the manuscript. They are minor and hopefully my suggestions improve the clarity of the paper. If anything is unclear, just let me know.

Comments as part of https://github.com/openjournals/joss-reviews/issues/3666 .

### Introduction

1. `Line 12: "Intro":` Suggest changing this to "Introduction" for a bit more formality.
2. `Line 13-14: "it is increasingly":` Phrasing of the first sentence is a bit awkward. Suggest something like: "Coastal zones host 40% of the world population with continued expected growth to be focused in least developed countries".
3. `Line 19: "risk to erosion":` Can add references to coastal monitoring programs, e.g. [Turner16](https://www.nature.com/articles/sdata201624), [Ruessink19](https://www.mdpi.com/2306-5729/4/2/73), [Ludka19](https://www.nature.com/articles/s41597-019-0167-6)
4. `Line 20: "Unmanned Aerial Vehicles":` The README defines UAVs as "Unoccupied Aerial Vehicles". I suggest changing 'Unmanned' to 'Unoccupied' to be consistent.
5. `Line 21: "emerging as the best platform":` I think "best" is a bit too definitive, as each platform has its pros and cons. Suggest changing to "emerging as an ideal platform" or something similar.
6. `Line 23: "at the mesoscale, a spatiotemporal resolution":` Should this be "at the mesoscale, and at a spatial temporal resolution"?
7. `Line 25: "researchers already use UAV-SfM to monitor beach dynamics":` Can you include a couple of example references?
8. `Line 26-27: "UAV-SfM technology is mature and reliable enough":` I think this wording is too strong and you could say instead "More recently, UAV-SfM technology is being increasingly used for wider-scale..."
9. `Line 28: "monitoring program mobilies more than":` Should this be "monitoring program has mobilized more than"
10. `Line 29-32:` Suggest splitting the last sentence of the paragraph into two, the word "which" is used twice.

### Statement of Need

1. `Line 39:` QGIS should be all caps. Can also consider adding a [citation](https://qgis.org/en/site/getinvolved/faq/index.html#how-to-cite-qgis).
2. `Line 42: "multitemporal DSMs is usally approached":` Should be "multitemporal DSMs are usually approached"
3. `Line 42: "dem of difference method":` Should be "DEM of difference method"
4. `Line 44: "from time to time":` Should be "between time intervals"
5. `Line 44: "two pre and post raster":` Remove "two" as it could be a little ambiguous - are the two pre rasters and two post rasters?
6. `Line 53:` Suggest the following replacement in the last sentence of the paragraph: "....beach wracks (macroalgae, woody debris), which requires time consuming manual processing to remove or filter to avoid biasing sediment volumetric computations."
7. `Line 76:` Define the BCDs acronym here, and perhaps a sentence or two saying what they are and why they could be useful? I think you already have some text in your jupyter notebooks you could use here.

**Comment by npucino** [comment]
Hi @chrisleaman , your suggestions greatly improved my manuscript, thanks a lot.

I implemented all your suggestions except for:

9. `Line 28: "monitoring program mobilies more than":` Should this be "monitoring program has mobilized more than".<br>
The program is still active and new datasets keep flowing in. I realised my wording misled you as I wrote that sentence using the past tense, so I now modified it into the present tense to make sure it is clear citizen scientists are still flying. And also replaced `mobilies `to `mobilises`.
Here is the snippet:
`For instance in Victoria (Australia), a citizen-science UAV-SfM monitoring program mobilises more than 150 volunteers to fly UAVs on 15 sites every six weeks during the last three years.`

6. `Line 23: "at the mesoscale, a spatiotemporal resolution":` Should this be "at the mesoscale, and at a spatial temporal resolution"?<br>
I used this wording instead: `at the mesoscale, which is an appropriate spatiotemporal scale for coastal managers to work with (Thom et al., 2018).

> here is the reference if interested: [Thom, B. G. et al. National sediment compartment framework for Australian coastal management. Ocean Coast. Manag. 154, 103–120 (2018)](https://linkinghub.elsevier.com/retrieve/pii/S0964569117306129)

Thanks again for this review!
Cheers
`
